packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -492,6 +487,17 @@ export class PipelineEditor extends React.Component<
     }
   }
 
+  /*
+   * Open node associated notebook
+   */
+  handleOpenNotebook(selectedNodes: any): void {
+    for (let i = 0; i < selectedNodes.length; i++) {
+      const path = this.canvasController.getNode(selectedNodes[i]).app_data
+        .filename;
```

Comment:
Could we please join this to the line above?  It reads so much easier and was that way prior to the refactor.

Reply:
this is probably a lint fix

Reply:
yes, lint... @ajbozarth maybe we could increase the line length :)

Reply:
FWIW, the backend uses a max-line length of 120.  IMO, limiting lines to 80 characters is too constraining, especially given terminals of the last **30 years**.  If I was using a  [VT100](https://upload.wikimedia.org/wikipedia/commons/9/99/DEC_VT100_terminal.jpg) again, yeah, lines > 80 characters would make things difficult to read.